backend/.env.example

```diff
@@ -1,2 +1,7 @@
 OIDC_CLIENT_ID=""
 OIDC_CLIENT_SECRET=""
```

Comment:
Should we add `CONSENT_REQUEST_TOKEN_ISSUER` also to this file? Would maybe make it more clear that this is something you need to configure.